### PR TITLE
refactoring of sign up spec flow

### PR DESCRIPTION
**Pull request by murad755**
Refactoring Sign Up spec flow

### CODE REVIEW

File: cypress/integration/nebula/TC01_signUp.spec.js [hunk 1]
```diff
@@ -1,311 +1,214 @@
 // <reference types="cypress" />
-import {getAccount}  from "../../support/Page_Objects/signUpPage.js";
+import {SignUpPage, signUpSelectors, signUpTexts} from "../../support/Page_Objects/signUpPage.js";
```
Comment (mikalai-okun-ft) [inline cypress/integration/nebula/TC01_signUp.spec.js:2]: Did you run the eslint/prettier?

**Review by mikalai-okun-ft — APPROVED** [review]

File: cypress/integration/nebula/TC01_signUp.spec.js (hunk 1 above)
Comment (murad755) [reply cypress/integration/nebula/TC01_signUp.spec.js:2]: I couldn't find prettier in a project, but I used eslint.

**Review by ManpreetChopraKaur — COMMENTED** [review]
@murad755 change "Kristina" email to
"techadmin" + randomChars(4)+ "@fundthrough.com"
change "saksham+" to "techadmin"

**Review by ManpreetChopraKaur — APPROVED** [review]

File: cypress/support/Helpers/common/button.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    cy.get('button').contains(value).parent().click()
+}
+
+export const checkButtonIsDisabled = (buttonName) => {
+    cy.get('button').contains(buttonName).parent().should('be.disabled')
+}
+
+export const checkButtonIsActive = (buttonName) => {
+    cy.get('button').contains(buttonName).parent().should('not.be.disabled')
+}
```
Comment (Praisy02) [inline cypress/support/Helpers/common/button.js:11]: Can we add a common function for getError webElement?

File: cypress/support/Page_Objects/signUpPage.js [hunk 3]
```diff
@@ -1,283 +1,358 @@
...
+    accountSetupFunding: 'Review the funding agreement',
+    accountSetupDeposit: 'Provide deposit information',
+    customerCallCheckbox: 'This is also what my customers call my business.'
+};
+
+export class SignUpPage {
+
+    visit() {
+        cy.visit('/')
+        cy.url().should('include', 'signup')
```
Comment (Praisy02) [inline cypress/support/Page_Objects/signUpPage.js:63]: can you use the endpoint in cy.visit('/signup') and change the baseurl to https://nebula-qa1.fundthrough.com

**Review by Praisy02 — CHANGES_REQUESTED** [review]

**Review by Praisy02 — APPROVED** [review]

**Pull request merged**